Readability.js

```diff
@@ -1745,7 +1746,9 @@ Readability.prototype = {
 
     // XXX implement _reduceNodeList?
     this._forEachNode(element.getElementsByTagName("a"), function(linkNode) {
-      linkLength += this._getInnerText(linkNode).length;
+      var href = linkNode.getAttribute("href");
+      var coefficient = href && href.match(this.REGEXPS.hashUrl) ? 0.2 : 1;
+      linkLength += this._getInnerText(linkNode).length * coefficient;
```

Comment:
Interesting - why not drop scoring for these links completely (ie why 0.2 rather than 0, or wrapping the `linkLength +=` statement in an if condition testing the href?

Reply:
I originally had that that way. But after testing edge cases and playing with `isList` detection it will I decided to add coefficient.
For example when you have an element that has mix of hash links and full links you want to be more sensitive then just considering hash links as plain text.

Reply:
After more testing I changed the `coefficient` to `0.3` so it sits between the `0.5` threshold for links and `0.2` for other content. That way it works really well with test cases and have only 1 false positive.